OpenTabletDriver/Output/OutputMode.cs

```diff
@@ -113,8 +113,7 @@ public InputDevice Tablet
         public virtual void Consume(IDeviceReport report)
         {
             if (report is IAbsolutePositionReport tabletReport)
-                if (Transform(tabletReport) is IAbsolutePositionReport transformedReport)
-                    report = transformedReport;
+                report = Transform(tabletReport);
 
             Emit?.Invoke(report);
```

Comment:
```suggestion
                report = Transform(tabletReport)!;

            if (report is not null)
                Emit?.Invoke(report);
```
Should not send `null` report to avoid future bugs where non-nullable `IDeviceReport report` is unexpectedly null for subsequent elements.